knowledge-base/kb-gridview-style-aggregate-results-in-group-header.md

```diff
@@ -0,0 +1,358 @@
+---
+title: Style aggregate results displayed in the GroupHeaderRow
+description: Update the style of aggregate function representation based on the result
+type: how-to
+page_title: Conditionally style aggregate results shown in the header of the group
+slug: kb-gridview-style-aggregate-results-in-group-header
+position: 0
+tags: aggregate, results, styling, conditionally, group, header
+ticketid: 1442680
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tr>
+		<td>Product</td>
+		<td>RadGridView for WPF</td>
+	</tr>
+</table>
+
+## Description
+
+How to change the style of the aggregates based on the results. 
+
+## Solution
+
+The approach would differ depending on the value of the [ColumnAggregatesAlignment]({%slug gridview-grouping-aggregates%}#align-aggregates-in-groupheaderrow) property of the RadGridView. If its value is __NoAlignment__ (which is the default), the aggregate results are shown in __GridViewAggregateResultCell__ elements. If its value is __NextToGroupKey__ or __BelowToGroupKey__, the results are displayed in __GridViewGroupHeaderCell__ elements. This article will show how you can conditionally style both elements. 
+
+__Example 1__ demonstrates a sample model and viewmodel that will be used for demonstration purposes. 
+
+#### __[C#] Example 1: The model and viewmodel__
```

Comment:
In order to keep the article as short as possible we can omit the caption of the code snippet and leave only the language - #### __[C#]__